pkg/capabilities/capabilities_test.go

```diff
@@ -18,17 +18,10 @@ package capabilities
 
 import (
 	"reflect"
-	"sync"
 	"testing"
 )
 
 func TestGet(t *testing.T) {
-	defer func() {
```

Comment:
leave this defer in place to leave the package in the same state when this test is done

Reply:
done! thanks @liggitt